travis/install_pg

```diff
@@ -4,13 +4,38 @@
 
 set -eux
 
-# always install postgresql-common
-packages="postgresql-common libedit-dev libpam0g-dev libselinux1-dev"
+# If USE_CUSTOM_PG_BUILD variable is not set, we will assume custom build is
+# not required. If custom build is not required, we will install PostgreSQL
+# via apt, otherwise we will compile from the source.
+if [ "${USE_CUSTOM_PG_BUILD-0}" = "0" ]; then
 
-# we set PGVERSION to 10x of the Citus version when testing Citus, so
-# only install PostgreSQL proper if it's 9.5 or lower
-if [ "${PGVERSION//./}" -le "96" ]; then
-  packages="$packages postgresql-$PGVERSION postgresql-server-dev-$PGVERSION"
-fi
+  # always install postgresql-common
+  packages="postgresql-common libedit-dev libpam0g-dev libselinux1-dev"
+
+  # we set PGVERSION to 10x of the Citus version when testing Citus, so
+  # only install PostgreSQL proper if it's 9.5 or lower
+  if [ "${PGVERSION//./}" -le "96" ]; then
+    packages="$packages postgresql-$PGVERSION postgresql-server-dev-$PGVERSION"
+  fi
+
+  sudo apt-get -y -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" install $packages
+else
 
-sudo apt-get -y -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" install $packages
+  # clone PostgreSQL 
+  cd ~
+  if [ "$PGVERSION" = "9.5" ]; then
+    git clone -b REL9_5_STABLE --depth 1 git://git.postgresql.org/git/postgresql.git
```

Comment:
Just do `git clone -b REL${PGVERSION//./_}` to replace the dot in-line. no need for conditionals.